tests/functional/tlsprofile/test_centralized_tlsprofile_configuration.py

```
@@ -0,0 +1,244 @@
+    """
+
+    @pytest.fixture(autouse=True)
+    def cleanup_tlsprofile(self, request):
+        tls = TLSProfile()
+
+        def _cleanup():
+            try:
+                if tls.is_tls_profile_available(silent=True):
+                    log.info("Teardown: deleting leftover ocs-tls-profile")
+                    tls.delete_tls_profile(wait=True, force=True)
+            except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)